owncloudData/src/androidTest/java/com/owncloud/android/data/sharing/shares/db/OCShareDaoTest.kt

```diff
@@ -77,33 +76,27 @@ class OCShareDaoTest {
     fun insertSharesFromDifferentFilesAndRead() {
         ocShareDao.insert(
```

Comment:
You could move this kind of calls to a `givenListOfShares` method to make the test more readable, like part of the `given, that, then` in tests... 🍻

Reply:
Well, probably that is out of this PR scope, here I have just replaced the mappers. If we need to refactor some test naming, I think that we could do it in another PR.